app/src/util/networks.ts

```diff
@@ -88,6 +89,15 @@ const networks: { [K in NetworkId]: Network } = {
   [networkIds.MAINNET]: {
     label: 'Mainnet',
     url: `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`,
+    alternativeUrls: [
+      {
+        rpcUrl: `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`,
+        name: 'Infura',
+      },
+      { rpcUrl: 'https://cloudflare-eth.com/', name: 'Cloudflare' },
+      { rpcUrl: 'https://blockscout.com', name: 'BlockScout' },
```

Comment:
blockscout mainnet endpoint should be `https://blockscout.com/poa/xdai/api/eth-rpc`

Reply:
awesome seems that i was targeting wrong url! Now it works!